dotcom-rendering/src/components/ArticleMeta.stories.tsx

```diff
@@ -89,17 +97,13 @@ export const ArticleStory = () => {
 		</Wrapper>
 	);
 };
+ArticleStory.storyName = 'Article';
 
-const appsFormat = {
-	display: ArticleDisplay.Standard,
-	design: ArticleDesign.Standard,
-	theme: Pillar.News,
-};
 export const ArticleAppsStory = () => {
 	return (
 		<Wrapper>
 			<ArticleMeta
-				format={appsFormat}
+				format={defaultFormat}
```

Comment:
It might make more sense to use the fact that the Story receives a `format`, now?

Applicable to all stories with decorators in this file.